java-frontend/src/main/java/org/sonar/java/se/checks/UnclosedResourcesCheck.java

```diff
@@ -36,27 +44,28 @@
 import org.sonar.plugins.java.api.tree.IdentifierTree;
 import org.sonar.plugins.java.api.tree.MemberSelectExpressionTree;
 import org.sonar.plugins.java.api.tree.MethodInvocationTree;
+import org.sonar.plugins.java.api.tree.MethodTree;
 import org.sonar.plugins.java.api.tree.NewClassTree;
 import org.sonar.plugins.java.api.tree.ReturnStatementTree;
 import org.sonar.plugins.java.api.tree.Tree;
 import org.sonar.plugins.java.api.tree.TryStatementTree;
 import org.sonar.plugins.java.api.tree.VariableTree;
 
-import javax.annotation.Nullable;
-
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-
 @Rule(key = "S2095")
 public class UnclosedResourcesCheck extends SECheck {
 
   private enum Status {
     OPENED, CLOSED
   }
 
+  private static final String DEFAULT_EXCLUDED_TYPES = "";
+  
+  @RuleProperty(
```

Comment:
I updated the description of the rule in order to include the parameter. Please follow the description from here for keys and description: [RSPEC-3591](https://jira.sonarsource.com/browse/RSPEC-3591)